src/components/include/media_manager/media_manager.h

```diff
@@ -36,16 +36,62 @@
 #include <string>
 #include "protocol/service_type.h"
 #include "media_manager/media_manager_settings.h"
+#include "utils/macro.h"  // for "DEPRECATED"
 namespace media_manager {
 
+/**
+ * \brief Describes different sampling options for PerformAudioPassThru.
+ */
+typedef enum {
+  /// Invalid sampling rate.
+  SR_INVALID = -1,
+  /// Sampling rate of 8000 Hz.
+  SR_8KHZ = 0,
+  /// Sampling rate of 16000 Hz.
+  SR_16KHZ = 1,
+  /// Sampling rate of 22050 Hz.
+  SR_22KHZ = 2,
+  /// Sampling rate of 44100 Hz.
+  SR_44KHZ = 3
+} SamplingRate;
+
+/**
+ * \brief Describes different quality options for PerformAudioPassThru.
+ */
+typedef enum {
+  /// Invalid audio quality.
+  ACQ_INVALID = -1,
+  /// Audio sample is 8 bits wide, unsigned.
+  ACQ_8_BIT = 0,
+  /// Audio sample is 16 bits wide, signed, and in little endian.
+  ACQ_16_BIT = 1
+} AudioCaptureQuality;
+
+/**
+ * \brief Describes different audio type options for PerformAudioPassThru.
```

Comment:
```suggestion
 * @brief Describes different audio type options for PerformAudioPassThru.
```

Reply:
Fixed in commit 5c9b72ecd09ab2f3427fb37ed716340992ecda0d.